dm/tests/openapi/run.sh

```diff
@@ -376,6 +381,19 @@ function run() {
 	test_relay
 
 	test_shard_task
+
+	export GO_FAILPOINTS="github.com/pingcap/tiflow/dm/dumpling/dumpSleepStage=return()"
```

Comment:
if you want to inject failpoint and test dump status in `test_noshard_task`, better to  open this failpoint in that case otherwise, this failpoint will affets other test case such as `test_multi_tasks` 

also remeber to use `export GO_FAILPOINTS="` to close failpoint and restart dm-worker to close failpoint after you test